docs/api-doc.md

```diff
@@ -0,0 +1,160 @@
+## Introduction
+
+Libsass wouldn't be much good without a way to interface with it. These interface documentations describe the various functions and data structures available to implementers. They are split up over three major components, which have all their own source files (plus some common functionality).
```

Comment:
Here's another spot where "LibSass" is a unique spelling (Libsass here) <-- again, super minor